.github/workflows/PR-5.x.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
 
   macOS-x64:
     uses: opencv/ci-gha-workflow/.github/workflows/OCV-PR-5.x-macOS-x86_64.yaml@main
+    timeout-minutes: 90
```

Comment:
@invarrow, sorry, that won't work. Please remove that line, I changed timeout here: https://github.com/opencv/ci-gha-workflow/pull/164.